hash_to_array.rb

```diff
@@ -0,0 +1,18 @@
+def hash_to_arrays(hash)
+    keys_array = hash.keys
+    values_array = hash.values
+    [keys_array, values_array]
+  end
```

Comment:
The assignment reads:

> Write a function that accepts the hash as a parameter **and prints out an array of keys and an array of values**.

So the puts statements are also in the wrong place here, but again it's not that important.

```suggestion
def print_hash(hash)
    keys_array = hash.keys
    values_array = hash.values
    puts "Keys array: #{keys_array}"
    puts "Values array: #{values_array}"
  end
```